src/wallet/wallet.cpp

```diff
@@ -3920,157 +4011,83 @@ bool CWallet::CreateTransaction(const std::vector<CRecipient>& vecSend, CWalletT
                     nChangePosInOut = -1;
                 }
 
-                // Fill vin
-                //
-                // Note how the sequence number is set to max()-1 so that the
-                // nLockTime set above actually works.
-                vecTxDSInTmp.clear();
-                for (const auto& coin : setCoins) {
-                    CTxIn txin = CTxIn(coin.outpoint,CScript(),
```

Comment:
@PastaPastaPasta look, it was here before. Now its 100 lines above

Reply:
Yeah... But git can be surprising good at resolving conflicts like this

Reply:
Come on, this method is already so far from upstream due to segwit, privatesend and now this bugfixing that this single line which is already 100 lines above the origin being a bit more like upstream won't bring any merge improvement, would it? This will probably just be a big conflict block no matter if a059633 is added or not 😅